### PR TITLE
Cleaned up readme, moving some bits to webrtc-uwp.github.io repo

### PR DESCRIPTION
**Pull request by jamescadd**
Updating the SDK documentation with some cleanup.  Some details moved to the pages repo, see this PR: https://github.com/webrtc-uwp/webrtc-uwp.github.io/pull/1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,35 +1,43 @@
...
-3) Now you can build WebRtc Windows Universla library and run PeerCC sample app.
+## Building the SDK
 
-====
+Several build options, including NuGet package creation, are available through command line scripts or Visual Studio.  For more information see the [webrtc-scripts](https://github.com/webrtc-uwp/webrtc-scripts) repo.  The simplest method to build the SDK and PeerCC sample in Visual Studio is described below.
+
+1. Open the solution `webrtc-uwp-sdk\webrtc\windows\solutions\WebRtc.Universal.sln` in Visual Studio
+2. Right click the `PeerConnectionClient.WebRtc` project and choose **Set as Startup Project**
+3. Choose processor architecture
+4. Hit F5
```
[the remark this answers is not in the document]
Comment (SergioOpenPeer) [reply README.md]: It looks like this is not necessary with the latest VS  15.9.7

File: README.md [hunk 2]
```diff
@@ -1,35 +1,43 @@
...
+| `webrtc-uwp-sdk\webrtc`  | WebRTC code - builds as Org.WebRtc.dll           |
+| `webrtc-uwp-sdk\docs`    | documentation                                    |
 
-How to Build:
+## Prerequisites
 
-WINDOWS BUILD :
-----------------------------
+- Visual Studio 2017 (latest tested version tested 15.9.7); Community Edition is supported
+- Windows SDK 17763 (available from [archive page](https://developer.microsoft.com/en-us/windows/downloads/sdk-archive))
```
[the remark this answers is not in the document]
Comment (jamescadd) [reply README.md]: Do the debugging tools need to be installed for that SDK version too?

[the remark this answers is not in the document]
Comment (jamescadd) [reply README.md]: Tentatively fixed

[the remark this answers is not in the document]
Comment (SergioOpenPeer) [reply README.md]: Yes. Debugging tools needs to be installed.

**Pull request merged**